# Values for the Sensor Thresholds
float64 thresholdLeft
float64 thresholdRight
float64 thresholdForward